package.json

```diff
@@ -9,12 +9,17 @@
     },
     "license": "BSD-3-Clause",
     "dependencies": {
+        "bootstrap": "3.3.7",
         "electron": "3.0.2",
         "electron-edge-js": "8.3.8",
         "electron-localshortcut": "3.1.0",
+        "gpii-handlebars": "1.1.4",
         "infusion": "3.0.0-dev.20190328T144119Z.ec44dbfab",
+        "markdown-it": "^8.4.2",
```

Comment:
What is this dependency for? I see you include it in captureTool/index.html but I don't see it is used

Reply:
Right, this was actually something I was tracking down when you were giving me tips about reloading the electron window.
gpii-handlebars includes a markdown component in a grade that gets subgraded (subclassed? subinfused?) by a number of things, and so will blow up if it's not available. I actually tried overriding it in my main component once, but it shows up in other renderer infrastructure things that get created as well.

https://github.com/GPII/gpii-handlebars/blob/master/src/js/client/renderer.js#L32-L34

Reply:
Please file this issue with @the-t-in-rtf

Reply:
I just confirmed that the required markdown plugin is a direct (non-dev) dependency of gpii-handlebars, but I do recall having to do something similar in other work that relied on gpii-handlebars. 

I also just realised that we never actually cut the full release of gpii-handlebars@2.1.0, I just took care of that.  I'd strongly suggest upgrading to 2.1.0 when you get the chance, as the 2.1.x line is where I'll fix the markdown dependency issue.

Reply:
Please do the gpii-handlebars update if you can - also, if this dependency still remains, please pin to an exact version

Reply:
bump

Reply:
Ok, updated to gpii-handlebars 2.1.0 and removed the markdown-it from packages.json.